src/Stratis.Bitcoin/P2P/PeerAddress.cs

```diff
@@ -162,6 +163,33 @@ public bool Handshaked
         [JsonIgnore]
         public DateTime? LastDiscoveredFrom { get; private set; }
 
+        /// <summary>
+        /// Determine whether the peer has been banned.
+        /// <para>
+        /// This is set in <see cref="PeerBanning"/>.
+        /// </para>
+        /// </summary>
+        [JsonProperty(PropertyName = "isbanned", NullValueHandling = NullValueHandling.Ignore)]
+        public bool? IsBanned { get; set; }
```

Comment:
I am not sure we need this property at all in store as it enough to have the `BanUntil ` to know if a peer is banned and for how long, how ever it might be useful to know when the ban was created.

So perhaps we only need to properties  `BanDate` and `BanUntil` and maybe also reason (but not sure yet about as this will be available in the logs if needed)

Reply:
Ok, I'll change to this.  As we have the banned reason in the logs I'll remove the BannedReason property.